app/services/transaction.service.js

```diff
@@ -73,28 +73,28 @@ const ledgerFindWithEvents = async function ledgerFindWithEvents (accountId, cha
 }
 
 const refund = function refundTransaction (accountId, chargeId, amount, refundAmountAvailable, userExternalId, userEmail, correlationId) {
-  return new Promise(function (resolve, reject) {
-    const payload = {
-      amount: amount,
-      refund_amount_available: refundAmountAvailable,
-      user_external_id: userExternalId,
-      user_email: userEmail
-    }
-
-    logger.log('info', 'Submitting a refund for a charge', {
-      'chargeId': chargeId,
-      'amount': amount,
-      'refundAmountAvailable': refundAmountAvailable,
-      'userExternalId': userExternalId
-    })
+  const payload = {
+    amount: amount,
+    refund_amount_available: refundAmountAvailable,
+    user_external_id: userExternalId,
+    user_email: userEmail
+  }
+
+  logger.log('info', 'Submitting a refund for a charge', {
```

Comment:
We've discussed how ideally we should be using standard logging keys here but we think there's a slightly bigger piece of work about making that easier in self service (potentially using a similar methodology to frontend or self service). 

That won't be part of this PR to keep it focussed.